docs/TEMPLATE.md

```diff
@@ -1,4 +1,9 @@
 ---
+title: The title of the post, page, or document
+description: A short description of the page's content
+image: URL to an image associated with the post, page, or document (e.g., /assets/page-pic.jpg)
+author: Page-, post-, or document-specific author information (see [Advanced usage](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/advanced-usage.md#author-information))
```

Comment:
Can we just remove `author`? If not, let's hard-code it to `Expensify`

Reply:
Yeah I was wondering if in the future we want to include the author of each article in the meta tags, but I think that's something we can bring to discussion once the articles are added. I'm going to remove it atm.